app/serializers/api/property_serializer.rb

```diff
@@ -1,4 +1,7 @@
 class Api::PropertySerializer < ActiveModel::Serializer
+  # TODO: stick to :presentation or :name not both because the are duplicate
```

Comment:
imo, code should never ever have TODOs. TODOs belong in backlogs. So I think this should be either 1. a comment about how the code works ":presentation or :name are duplicates and could be one property only" or 2. an issue in GH

Reply:
Ups, sorry my mistake. I left it there to turn it into something else. I share the opinion about TODOs

Reply:
I created https://github.com/openfoodfoundation/openfoodnetwork/issues/3313